samples/templates/express-embedded-auth-with-sdk/web-server/assets/js/webauthn.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2018, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+// Utils to convert beyween string and ArrayBuffer
+const CryptoUtil = {
```

Comment:
We can include these functions in the SDK layer, the widget repo may have similar implementation & tests, we can keep it in the auth-js repo, then let widget use the functions from auth-js import (follow up step).

Reply:
Moved that utils to `OktaAuth.crypto` (and exported `crypto`), added tests.